tests/Fixtures/Integration/priority/increment_style,standardize_increment.test

```diff
@@ -0,0 +1,11 @@
+--TEST--
```

Comment:
Please add a test case to [`FixerFactoryTest::testFixersPriority()`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.10.0/tests/AutoReview/FixerFactoryTest.php#L60).

Reply:
Ah didn't realise that had to be added! Would be helpful to have that in the fixer cookbook

Reply:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3518